tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -99,14 +101,15 @@ def is_model_file_to_convert(file_path: pathlib.Path):
             (output_dir / relative_model_path).parent.mkdir(parents=True, exist_ok=True)
 
             ort_target_path = (output_dir / relative_model_path).with_suffix(
-                _optimization_suffix(optimization_style, ".ort"))
+                _optimization_suffix(optimization_level_str, optimization_style, ".ort"))
 
```

Comment:
Can we skip adding the level to the filename if it's the default value? 

Otherwise I expect we need to update other scripts/documentation given the generated suffix is changing.